### ISSUE TITLE
Relog issue

### ISSUE THREAD

**Issue by vcsifirbir** [HerrEurobeat/steam-idler#6]
Hey,
When lost connection to Steam, asks Steam Guard Code for relog. But when I close and reopen , it logs in without asking for Steam Guard Code with a token.

![image](https://user-images.githubusercontent.com/122023113/228408022-5ed50d40-51cf-419e-9798-cda095440395.png)

**Comment by HerrEurobeat** [comment]
Hey, thanks for opening an issue!  
I'm going to investigate the issue tomorrow and will let you know when it's fixed.

**Comment by HerrEurobeat** [comment]
Alright, found the issue! This bug only appears when using accounts with mobile steam guard (without shared_secret), which is a combination I hadn't tested back when I implemented the relog system for sessions.  
It occurs because I forgot to pass the token to logOn() on relog and instead used credentials - which tries to use the old login flow and fails. Restarting "fixed" this as I used the token on initial login as expected.  
  
I'll comment again when version 1.5 is out.
